doc/contributing/advocacy-ambasador-program.md

```diff
@@ -0,0 +1,93 @@
+# Advocacy Ambassador program
+
+Node.js aims to support and encourage developers and developer
+advocates to promote Node.js as a project and to help communicate
+key messages to their constituencies.
+
+The ambassador program does that by:
+
+* Maintaining an up-to-date set of messages that it would like ambassadors
+  to promote. These will be documented in the nodejs/ambassadors repository.
+* Maintaining an up-to-date set of topics on which additional content
+  would benefit the community. These will be documented in the
+  nodejs/ambassadors repository.
+* Providing a space for collaborators to share new topics/features they
+  are working on, when they would be happy for ambassadors to help share and get
+  early feedback.
+* Helping ambassadors promote content that aligns with the
+  messages and topics defined through social media and other channels.
```

Comment:
The ambassador program helps ambassadors promote content? Is that what is intended by this sentence? If not, it needs a rewrite. If so, it would be useful to understand how the ambassador program helps ambassadors promote content. It seems like the purpose actually goes the other way: The ambassador program helps Node.js promote content via ambassadors recruited for that purpose.

Reply:
Maybe this is what is intended?

```suggestion
* Promoting relevant ambassador-authored content
  through social media and other channels.
```

Reply:
I don't think "relevant" is a good repalcement for " that aligns with the
  messages and topics defined".  I'll ake out the last part "through social media and other channels" in case the problem was the confusion in the wording on how they were definfed.